data/campaigns/Heir_To_The_Throne/scenarios/05b_Isle_of_the_Damned.cfg

```diff
@@ -57,11 +57,8 @@
 
             type="Elvish Sylph,Great Mage," +
```

Comment:
I played this scenario recently and I remember that I cannot recall a single elf here. Only merfolk recalls and outlaw recruits...so why are the elves even here in the advisor creation?

Reply:
They are given the role on prestart, before they are disallowed, that specific unit will be the one talking on behalf of the whole ship.